app/controllers/api/v1/display_ads_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1
+    class DisplayAdsController < ApiController
+      before_action :authenticate!
```

Comment:
**Question:** Perhaps this should be `authenticate_with_api_key!` instead, given that we're making this endpoint available via API key. At least that's what I understand from reading issue that this would be the only auth option requested for this endpoint. What do you think about this? Any reason to keep it this way?

Reply:
I... _think_ that's right? I'm a bit confused about the way authentication works in our APIs, so I was mostly cargo-culting here.